core/src/validator.rs

```diff
@@ -1589,6 +1618,7 @@ fn load_blockstore(
             transaction_history_services
                 .cache_block_meta_sender
                 .as_ref(),
+            entry_notification_sender.as_ref(),
```

Comment:
Does this mean on restart, when replaying entries from blockstore, we will potentially send items to geyser we have already sent from before the restart? I'm guessing geyser handles this?

Reply:
Yes, I think that's right. This is mirroring the approach of transaction geyser notifications.
The geyser manager doesn't care one way or the other, but I think it's incumbent on a plugin to handle repeat notifications.